specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -100,63 +103,96 @@ namespace EventGrid {
   }
 
   @doc("The result of the Release operation.")
   model ReleaseResult {
-    @doc("Array of LockToken values for failed cloud events. Each LockToken includes the lock token value along with the related error information (namely, the error code and description).")
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
     failedLockTokens: FailedLockToken[];
 
-    @doc("Array of lock tokens values for the successfully released cloud events.")
+    @doc("Array of lock tokens for the successfully released cloud events.")
     succeededLockTokens: string[];
   }
 
   @doc("The result of the Reject operation.")
   model RejectResult {
-    @doc("Array of LockToken values for failed cloud events. Each LockToken includes the lock token value along with the related error information (namely, the error code and description).")
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
     failedLockTokens: FailedLockToken[];
 
-    @doc("Array of lock tokens values for the successfully rejected cloud events.")
+    @doc("Array of lock tokens for the successfully rejected cloud events.")
     succeededLockTokens: string[];
   }
 
-  @doc("Array of lock token strings for the corresponding received Cloud Events to be released.")
+  @doc("The result of the RenewLock operation.")
+  model RenewCloudEventLocksResult {
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
+    failedLockTokens: FailedLockToken[];
+
+    @doc("Array of lock tokens for the successfully renewed locks.")
+    succeededLockTokens: string[];
+  }
+
+  @doc("Array of lock tokens for the corresponding received Cloud Events to be released.")
   model ReleaseOptions {
-    @doc("String array of lock tokens.")
+    @doc("Array of lock tokens.")
     lockTokens: string[];
   }
 
-  @doc("Array of lock token strings for the corresponding received Cloud Events to be acknowledged.")
+  @doc("Array of lock tokens for the corresponding received Cloud Events to be acknowledged.")
   model AcknowledgeOptions {
-    @doc("String array of lock tokens.")
+    @doc("Array of lock tokens.")
     lockTokens: string[];
   }
 
-  @doc("Array of lock token strings for the corresponding received Cloud Events to be rejected.")
+  @doc("Array of lock tokens for the corresponding received Cloud Events to be rejected.")
   model RejectOptions {
-    @doc("String array of lock tokens.")
+    @doc("Array of lock tokens.")
+    lockTokens: string[];
+  }
+
+  @doc("Array of lock tokens for the corresponding received Cloud Events to be renewed.")
+  model RenewLockOptions {
```

Comment:
Is it intentional that `RejectOptions` and `RenewLockOptions` are separate models? Are they expected to diverge over time?

Reply:
As discussed in the meeting, right now they are same but we don't know if they will diverge or not, so keeping them different.